winui/Shadow/Customization.md

```diff
@@ -128,3 +128,61 @@ shadow.Content = button;
 {% endtabs %}
 
 ![Positioning in WinUI Shadow control](Shadow_images/winui_shadow_offset.png)
+
+## Enable or Disable the shadow
+
+The enable or disable the shadow can be customized using the `EnableShadow` property. The default value of the `EnableShadow` property is true.
```

Comment:
@Priyanka-Vijayasankar The Shadow control provides support to enable or disable the shadow effect by using the `EnableShadow` property. The default value of the `EnableShadow` property is true.

Reply:
Changed the content as mentioned above.